hack/gpg-wrapper.sh

```diff
@@ -1,19 +1,18 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
any particular reason to change the shebang on this file only? 
I see there are some other scripts which still use `#!/bin/sh`.  My knowledge of bash scripts is limited, unless the schebang change is specific to this file, could consider standardizing it across all scripts?

Reply:
Hmmm, maybe something I did during the refact, but it is not necessary for the current shape of the script - good catch! Let me revert this to keep the PR focused.

Regarding replacing sh with bash everywhere, I am all for it!